src/uiSelectController.js

```diff
@@ -325,9 +330,9 @@ uis.controller('uiSelectCtrl',
     var isDisabled = false;
     var item;
 
-    if (itemIndex >= 0 && !angular.isUndefined(ctrl.disableChoiceExpression)) {
+    if (itemIndex >= 0 && (!angular.isUndefined(ctrl.disableChoiceExpression) || ctrl.multiple)) {
```

Comment:
Actually the parentheses already disambiguate the precedence. The current code is shown below, with the matching middle parentheses spaced out for emphasis:

    if (itemIndex >= 0 &&   (  !angular.isUndefined(ctrl.disableChoiceExpression) || ctrl.multiple  )   ) {

Those parentheses ensure that the `||` in the middle takes precedence. The suggestions made above would result in double-parentheses on the same expression (the disjunction in the first example; the entire condition in the second example).